### PR TITLE
[docstring] Fix docstring for `CodeLlamaTokenizerFast`

### PR DESCRIPTION
**Pull request by Bojun-Feng**
# What does this PR do?

Fixes #26638

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by Bojun-Feng** [comment]
On Mac with M1 Chip, I received the following error when running `pip install -e ".[dev]"`
```ERROR: Could not find a version that satisfies the requirement tensorflow-text<2.15; extra == "dev" (from transformers[dev]) (from versions: none)
ERROR: No matching distribution found for tensorflow-text<2.15; extra == "dev"```

**Comment by Bojun-Feng** [comment]
@ydshieh 

Doc strings of `CodeLlamaTokenizerFast` seems to be properly updated, and this PR is ready to merge.

However, the above issues were a bit confusing and prevented me from updating the doc strings of `CodeLlamaTokenizer`. Am I setting up the dev environment incorrectly? Any information would be much appreciated!

**Comment by Bojun-Feng** [comment]
Update on issues encountered: `pip install -e ".[dev-torch]"` solved the issue for me.

**Review by ydshieh — COMMENTED** [review]
Very nice, thank you for the contribution 🔥 💯 .

I have one question and needs my colleague to confirm.

**Comment by ydshieh** [comment]
Regarding environment, I am not using Mac, and it seems tensorflow related stuffs are in the way (`tensorflow-text`).

Do you have `tensorflow` and/or `tensorflow-text` installed? What are their versions?

**Comment by Bojun-Feng** [comment]
> Regarding environment, I am not using Mac, and it seems tensorflow related stuffs are in the way (`tensorflow-text`).
> 
> Do you have `tensorflow` and/or `tensorflow-text` installed? What are their versions?

I have `tensorflow` at version `2.14.0`. I do not have `tensorflow-text`.

**Comment by ydshieh** [comment]
Maybe you can remove the 3 places of `tensorflow-text` in the file `setup.py` (do not commit) and  try install with `.[dev]` again, and see if this could resolve issue. But we can move on to merge this PR just with `CodeLlamaTokenizerFast`. We can leave `CodeLlamaTokenizer` for another PR.

**Comment by Bojun-Feng** [comment]
> Maybe you can remove the 3 places of `tensorflow-text` in the file `setup.py` (do not commit) and try install with `.[dev]` again, and see if this could resolve issue. But we can move on to merge this PR just with `CodeLlamaTokenizerFast`. We can leave `CodeLlamaTokenizer` for another PR.

Let's just merge it. I can create another PR for `CodeLlamaTokenizer`.

File: src/transformers/models/code_llama/tokenization_code_llama_fast.py [hunk 1]
```diff
@@ -75,37 +75,39 @@ class CodeLlamaTokenizerFast(PreTrainedTokenizerFast):
...
         middle_token (`str`, *optional*, defaults to `"▁<MID>"`):
             Middle token used for infilling.
+        suffix_token (`str`, *optional*, defaults to `"▁<SUF>"`):
+            Suffix token used for infilling.
         eot_token (`str`, *optional*, defaults to `"▁<EOT>"`):
             End of text token used for infilling.
         fill_token (`str`, *optional*, defaults to `"<FILL_ME>"`):
             The token used to split the input between the prefix and suffix.
-        suffix_first (`bool`, *optional*, default to `False`):
-            Whether the input prompt and suffix should be formatted with the suffix first.
```
[the remark this answers is not in the document]
Comment (ArthurZucker) [reply src/transformers/models/code_llama/tokenization_code_llama_fast.py:105]: It should and we should use it when setting the filling (use `self.suffix_first if suffix_first is None else suffix_first`)

**Review by ArthurZucker — COMMENTED** [review]
I can open the follow up PR to fix the suffix first issue!

**Comment by ArthurZucker** [comment]
Could you resolve the conflicts so that I can approve the pR? 😉

**Comment by Bojun-Feng** [comment]
> Could you resolve the conflicts so that I can approve the pR? 😉

I am a bit unsure what you mean by conflict. I have resolved the conversation, if that is what you meant. Please let me know if you would like me to add back the suffix_first parameter.

**Comment by ydshieh** [comment]
@Bojun-Feng 

Check `This branch has conflicts that must be resolved` below. It has conflict to the main branch as the (long) list of `OBJECTS_TO_IGNORE` is updated frequently due to the event.

**Comment by Bojun-Feng** [comment]
> @Bojun-Feng
> 
> Check `This branch has conflicts that must be resolved` below. It has conflict to the main branch as the (long) list of `OBJECTS_TO_IGNORE` is updated frequently due to the event.

Conflict resolved, thank you for the clarification!

**Review by ArthurZucker — APPROVED** [review]
Thanks for fixing 😉

**Pull request merged**